src/test/java/org/fcrepo/importexport/importer/ImporterTest.java

```diff
@@ -56,36 +56,34 @@
     private URI pairtreeURI;
     private URI finalContainerURI;
     private File binaryFilesDir;
-
     private FcrepoResponse conResponse;
     private PutBuilder binBuilder;
-
```

Comment:
This is very minor, but for readability, please add an empty line before `@Before`.